docs/user/tutorial/60-admin.rst

```diff
@@ -44,12 +40,33 @@ Add an image to the Filer
 
 .. todo:: write up steps properly
 
+.. image:: images/new_folder_button.png
+   :alt: The new folder button
+   :width: 20%
```

Comment:
Please use actual pixel sizes for buttons, not % (just for future reference, I will change this).